### PR TITLE
optimize: change the groupId to org.apache.seata

### PR DESCRIPTION
**Pull request by funky-eyes**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,15 +91,15 @@ Depending on the scenario, choose one of the two dependencies: `io.seata:seata-a
 <dependencies>
 <!--dependencies for non-SpringBoot application framework-->
   <dependency>
-    <groupId>io.seata</groupId>
+    <groupId>org.apache.seata</groupId>
```
Comment (slievrly) [inline README.md]: It is recommended to revert readme temporarily.

Comment (funky-eyes) [inline README.md]:
```suggestion
    <groupId>io.seata</groupId>
```

File: README.md [hunk 2]
```diff
@@ -91,15 +91,15 @@ Depending on the scenario, choose one of the two dependencies: `io.seata:seata-a
...
     <artifactId>seata-all</artifactId>
     <version>${seata.version}</version>
   </dependency>
 
 <!--If your project base on `Spring Boot`, you can directly use the following dependencies-->
 <!--Notice: `seata-spring-boot-starter` has already included `seata-all` dependency-->
   <dependency>
-    <groupId>io.seata</groupId>
+    <groupId>org.apache.seata</groupId>
```
Comment (funky-eyes) [inline README.md]:
```suggestion
    <groupId>io.seata</groupId>
```

File: README.md (hunk 1 above)
Comment (funky-eyes) [reply README.md]: done

**Review by slievrly — APPROVED** [review]
LGTM

**Comment by xingfudeshi** [comment]
Pls change the package for mock server module too.

**Comment by funky-eyes** [comment]
> Pls change the package for mock server module too.

done

**Pull request merged**